packages/cli/src/utils/compiler.ts

```diff
@@ -38,3 +44,42 @@ export const compile = async (projectPath: string, entryFile: string, outputName
             },
         },
     });
+
+export const compilePlatformApp = async ({ outputName }: CompilerOptions) => {
```

Comment:
Please add tests for this, it can be quite dumb but at least we would have a baseline to start